### PR TITLE
Fix/manifests

### PR DESCRIPTION
**Pull request by eumel8**
fixes: https://github.com/eumel8/cosignwebhook/issues/37

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1 @@
 cosignwebhook
-chart/caas-values.yaml
-
-# the keypair used for test-signing of the webhook
-*.key
-*.pub
```
Comment (puffitos) [inline .gitignore:6]: Please do not remove those files, as keys are generated for the e2e tests.

File: manifests/manifest.yaml [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
...WjdKWFhMdk1sNkZXSWc5MktOSDRxdVpFTXV1TFJpWQphbUNwaFFqaU1heWxuZmRKdHB5ZXVrbGtTRm5waENuTzZQYjVzc0p4WkhQV3JpbU9KTVdUa1UwL2N5MVFwUU9uCkt1K0pGYnlzNStWQjlJNSs2ZEY2KzhEOVlLZEUwSTIzUVNKT0c4QWNIU0svNHdVd1NoZkVBUVVDZ1lFQTRBRWMKWDhIWHRaeGYyQzcyNjU2Z0NjdmNRVXg0NEw0NFFORW5HK0t2dWlhVXNqTG13RE9meisrVFAxdEFweVZ0UElmTAp1VXNGQkRzeFIvT3Q2bHphR2NaYWlBYktUSzZoMXRmS2w1UDRXcFEyNFZtVXg1V2QwWXUwbTNsY0tVMWJKcVV0CmVCdHJWSXI2VnRUWTJ0UW5oMlZoYjBKWlRvVDAxY1A4SWdJQStYRUNnWUVBZy80eXVTOTZkczkyaDhseFh4YkMKZVlkTmluQUlkMUwxVnNiaDJoalRmTi9WOXNWMHJrMHRVQkRjWGxScTNYSUN6cVNFc3FOWG0wVnBRVUk1Z2l0aAoyWkVBbUV6K0pWdkx0cDdTeUZhckswVUJWRzZIazJST0VrdnpjUkZJY1lqQkt1ZXR5VjRZUFFjVEh1am02ODdqCnJGSWlINThEUW5aSFZjd3NNMXJpRFMwQ2dZRUExcS9aM09Dd2Q4dStoRGhHeVAzSGpud0d4Y0tnRzIzVUJqTXUKazJoSVdWVytyNmRmUnBoenMxdCt1WHNnMU5GbkIxdHk0a3pIS3czOEptU2lxM3FBUWJHOGRLQmxaT2IrRHlYVQpja0FNQit3RFNTbU5FVmdRNEpOUDdxNTZxTThON0ZUazVqY0pRQWJPOVVlYjZ1Ym9WN2pQa0UxN3dHaG5LelZoCndjby8ydEVDZ1lCSkZIQTBIMFJZOGJERjBlcVVNeGVyWnNBM1lOY0RZS2JlMFE0dmJEbENLa3A3SFJRU3Q0Ri8KbTJSNThHemNzWU85VXNFbHhZTnNDdEJNVFMySFF0KytVbjloUGxrVWtqbjQ1dGtzUWVleDR2cW1FZkN2dmZPZQpKVWdRTi8vQ2JjdDJ5QTVNeVJuZEdrcEJBd0UyeUR2NlA2Skczai91dHVmRW1pVlJiNm5CU3c9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```
Comment (puffitos) [inline manifests/manifest.yaml:9]: Isn't this file generated with the generate-certs.sh? Should we really include it, even if it's a one-off cert-key combo?

File: manifests/manifest.yaml [hunk 3]
```diff
@@ -148,7 +156,7 @@ webhooks:
...
...W9DZ2dFQkFMbStSZXN5bWxVZlRpNW5lY0hRY2EwdTE2THIKTFExZkt3SUhJbVpscDB5OG83ZXd1UE55U1dwZ3lEOHFzVUlXeE1Wa3huQjM2ekdrTDloK0h2ekQ4RjF2ckRydApXcEF1ek5ieEwrc1pvd3ZZcjVRZWE3UXNSTDczVnovZncxWXVSeVk0Umg2elo4Rm1rTCswZjc1RWZpVzBLdjhiCnJwdHVucUtPdm9ERVVkL2F3R25ySXh4MC9KTWlIR091emVwWXlGWEtmMTRyOTlaMTQyMHRvdy9SUzZjT0t6dXEKSHlMTzJaeGJaQmtpeUY1TUVZbXk3L3JicU1kd29Ud29KOVJqeWVTQnlrbUwzbTQxQm9xeUIzckRBMkZObDlpdwpUc0JOTUJUOU1GYlhtWGp6aUZZeDZUU3lMRjQxODFQQmx2Sm5yeDBCMHlqbjduR0lmTGx4ZVdDV3NtTUNBd0VBCkFhTmhNRjh3RGdZRFZSMFBBUUgvQkFRREFnS2tNQjBHQTFVZEpRUVdNQlFHQ0NzR0FRVUZCd01CQmdnckJnRUYKQlFjREFqQVBCZ05WSFJNQkFmOEVCVEFEQVFIL01CMEdBMVVkRGdRV0JCU3NGUFU0OEhKNmdVVDF6aE9GQUt3awpXRGNVWVRBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQUxON2FFbUtMVGpRbncranU0VkViZjJ4OWFtNEdlSmhOCjhobHVxZFMvdENYNHJmU0w1R1pWNGxFZ0lpSXNBWm9hOW9TRmU4bERxZTNhYXZIMnBOUHBRK3lHUldpdjlKUUUKZnJ0MkpzNC8xUlUvWHpiL3pWNHBqT3VXSCtwY29acGFVY20rc1JYYjFQeG5sMjlZYVdkdDRRdHhFNHJKVDlVSQp2U3dHWG1FaTR6MXpTWlNGRFh0Y24vQkFpMzZ2UHpnWDZxcG0vajhSSEFIK21mWXhZcExPQ2RpdWVmTkZLajRlCmtzU01RZjI4NW4rRDNWVUhlRHNFb2JQWDk5enIrOW1PdStvN044OVVCUXU1VzJhNFJIMkxmaFRWSUYzS3h1T3gKRXB0c1JXY0Z6OHRMcnlHOTBNcXBkU3JmZ0ljZXdNalhyV25RMi9TOVkraXNMUENGU3FPRGpRPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo="
```
Comment (puffitos) [inline manifests/manifest.yaml:159]: Same is valid for the caBundle - as it's patched via the generate script, we deploy it with an empty value and just patch it afterwards.

File: manifests/manifest.yaml (hunk 2 above)
Comment (eumel8) [reply manifests/manifest.yaml:9]: indeed, I would like to have a one-liner to install the webhook. The generate script is an additional step and not so failure-tolerant.  The user can end up with different ca and client cert, which will be confusing. I've added an extra warning now.

**Review by puffitos — APPROVED** [review]

**Pull request merged**